README.md

```diff
@@ -48,9 +48,22 @@ This series of events will help you develop SaaS applications that can:
 
 Description: Hands-on walkthrough modernizing a .NET app.
 
-In this session you will learn:
-
-- TBC
+In this session you will learn the steps to modernize a legacy on-premises application, including upgrading and migrating the database to Azure and updating the application to take advantage of serverless and cloud services. Azure services and related products includes:
```

Comment:
Nitpick: Should we change "cloud services" to "PaaS" or "App services" because "Cloud Services" is the name of a service that is being "deprecated". https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-choose-me

Nitpick: "Azure services and related products includes" -> "Azure services and related products include"

Reply:
Thanks @PlagueHO ! Cloud services refers to the various services (including Key Vault, Cognitive Services etc.) that is not exclusively within the app deployment realm. I will imagine life will be quite hard if we cannot address cloud services for what it is because of a service deprecation?

As for include and includes, I did check this actually, had the same inkling:

https://english.stackexchange.com/questions/45616/menu-of-services-include-or-includes/45676

So, in this case, I am referring to Azure as the singular object, similar to menu in the example above"?

Reply:
Haha all good then 😀